packages/web/hooks/use-overridable-state.ts

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react";
```

Comment:
I'm somewhat hesitant about renaming this hook to overridable, as it doesn't precisely convey its functionality. In my opinion, the term 'overridable' is quite broad; for instance, `useState` could technically be considered overridable as well. I believe keeping it as `controllable` is a more suitable name, given that the concept of controllable/uncontrollable is well recognized within the React ecosystem. This term shows that the hook's behavior can be switched between uncontrolled and controlled modes as required